### ISSUE TITLE
swapped monitors

### ISSUE THREAD

**Issue by tamis-laan** [baskerville/bspwm#1424]
I'm a bit confused, I'm using the following setup for my monitors:
``` bash
# Set HDMI monitor
bspc monitor HDMI-A-0 -d 5 6 7 8

# Set laptop monitor
bspc monitor eDP -d 1 2 3 4
```

I would expect desktops 1-4 to go to my laptop screen and 5-8 to go to my hdmi screen. However it's the exact opposite!? Is this a bug or am I missing something?

**Comment by seVenVo1d** [comment]
I don't think it is a bug. I have found two reddit posts, which might be helpful to your situation. In this [reddit post.](https://www.reddit.com/r/bspwm/comments/i8mmmv/how_to_set_up_two_monitors_properly/) it says

>Bspwm doesn’t care what you name your desktops. The fact that you named them I II III V etc. has no meaning other than that’s the string associated with that desktop. So when you use that sxhkd keybinding, you’re actually focusing desktops by their index, not their name. The first monitor will contain the first 5 desktops by index.

You can also look at [this one](https://www.reddit.com/r/bspwm/comments/q15ckk/dual_monitor_setup_not_working/) 

>If I remember correctly, the most important thing in bspwm is that you've FIRST to tell bspwm the correct order of your displays, with the command:

    bspc wm -O DP-0 DP-7

>I'm not using bspwm anymore these days but I clearly remember that I had to run this command in my bspwmrc BEFORE assigning workspaces to monitors, so play around with it and see if it solves your problem

**Comment by yqlbu** [comment]
Man, you save my life. I spent tons of time debugging this desktop order issue. Nice to have the solution here. Thanks!
